src/openrct2/object/ObjectList.cpp

```diff
@@ -33,7 +33,7 @@ static constexpr std::array<int32_t, EnumValue(ObjectType::count)> kObjectEntryG
     kMaxSceneryGroupObjects, // scenery sets
     kMaxParkEntranceObjects, // park entrance
     kMaxWaterObjects,        // water
-    kMaxScenarioTextObjects, // scenario text
+    kMaxScenarioMetaObjects, // scenario text
```

Comment:
```suggestion
    kMaxScenarioMetaObjects, // scenario meta
```

Reply:
Done. I've found replaced three instances of 'scenario text' in comments.